### DISCUSSION TITLE
10.8.0-beta3

### DISCUSSION THREAD

**Discussion by pdulvp** [pdulvp/jellyfin-qnap#19] [Announcements]
Version based on: `jellyfin-server_10.8.0-beta3` `jellyfin-web_10.8.0-beta3` `jellyfin-ffmpeg5_5.0.1-4`

**Improvements on Jellyfin**
* [jellyfin 10.8.0-beta3](https://github.com/jellyfin/jellyfin/releases/tag/v10.8.0-beta3)
* [jellyfin-ffmpeg 5.0.1-4](https://github.com/jellyfin/jellyfin-ffmpeg/releases/tag/v5.0.1-4)

**Additional improvements on QNAP packaging**
* Compatibility with QTS 5.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/pdulvp/jellyfin-qnap/releases/tag/10.8.0-beta3-fixes17'>10.8.0-beta3</a>.</em>

**Comment by nzapodege** [comment]
Wow ... the problem of the full cpu is solved with this new version. Thanks

**Reply by pdulvp** [reply]
Hi,
Hope it's ok. I haven't fix something specific about it.
Maybe If you updated the firmware to 5.0.1 fixed something. Will see ;)

**Reply by nzapodege** [reply]
The problem is that with 10.8.0b3 the player changes settings playing in 4k and consequently the cpu (on low quality films) goes to full

**Reply by nzapodege** [reply]
A good alternative is to use the codec "MPV" player
